package.json

```diff
@@ -52,11 +52,17 @@
     "firebase": "^3.4.1"
   },
   "devDependencies": {
+    "babelify": "6.4.0",
     "broccoli-asset-rev": "^2.4.2",
+    "browserify": "10.2.6",
+    "browserify-shim": "3.8.12",
+    "chai": "3.5.0",
+    "del": "1.2.0",
```

Comment:
Keep the deps sorted with all the non ember-cli ones at the bottom and a blank line separating them. it makes it much easier to upgrade ember-cli each time.

```
    "ember-try": "^0.2.2",
    "loader.js": "^4.0.1", // ^^ ember-cli stuff

    "babelify": "6.4.0",
    "browserify-shim": "3.8.12",
    "browserify": "10.2.6",
    "del": "1.2.0",
    "ember-cli-fastboot": "1.0.0-beta.9",
    "ember-fastboot-addon-tests": "0.1.0",
    "ember-sinon": "0.5.1",
    "glob": "5.0.13",
    "gulp": "3.9.1",
    "gulp-concat": "2.6.0",
```